File: for-battle.go
package main

import "log"
import "time"

func main() {
    start := time.Now()

	result := 0

	for i := 0; i < 1000000001; i++ {
		result += i
	}

	time_spent := time.Since(start)
	log.Printf("%s", time_spent)
}

File: for-battle.rs
fn main() {
    use std::time::Instant;
    let start = Instant::now();

    let mut result: u128 = 0;

    for i in 0..1000000001 {
        result += i;
    }

    let time_spent = start.elapsed();
    println!("{}", result);
    println!("{:.2?}", time_spent);
}